international-development-policy-journal.csl

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>International Development Policy Journal</title>
+    <id>http://www.zotero.org/styles/international-development-policy-journal</id>
+    <link href="http://www.zotero.org/styles/international-development-policy-journal" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-cite-them-right" rel="template"/>
+    <link href="https://poldev.revues.org/2085?file=1" rel="documentation"/>
+    <author>
+      <name>Catherine Brendow</name>
+      <email>catherine.brendow@graduateinstitute.ch</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <issn>1663-9375</issn>
```

Comment:
@cathbrend, can you add the electronic ISSN?

(add the line `<eissn>1663-9391</eissn>`)